mesheryctl/cmd/perf.go

```diff
@@ -86,7 +91,18 @@ var perfCmd = &cobra.Command{
 			log.Print(perfDetails)
 			return
 		}
+		if filePath != "" {
+			var t models.BenchmarkSpec
```

Comment:
A nit: where we can, it may serve us well to use the term PerformanceSpec as opposed to BenchmarkSpec. In doing so, we'll expand the implicit description of this Spec's applicability to use cases beyond benchmarking to include a number of other performance-related concerns.

Reply:
Done!

Reply:
This will be breaking change for those importing meshery through go though... But this migration is necessary, as it would be better to move PerformanceSpec go file in SMPS repository and have symlink present here only.